### ISSUE TITLE
Kaptain music

### ISSUE THREAD

**Issue by STVGecko** [OhMyGuus/I-Dont-Care-About-Cookies#11]
In this website, https://www.kaptainmusic.com/

There is a pop-up in the home page :
![Kaptain Music](https://user-images.githubusercontent.com/71868745/190723623-713ad409-413f-41d3-a2e0-2a20c01242bd.png)

**Comment by krystian3w** [comment]
For uBlock Origin MV2 and AdGuard MV2:

```
kaptainmusic.com##div[class*="CookieCategoryWrapper"]:upward(2)
```

**Comment by STVGecko** [comment]
> For uBlock Origin MV2 and AdGuard MV2:
> 
> ```
> kaptainmusic.com##div[class*="CookieCategoryWrapper"]:upward(2)
> ```
> 
> Pop-up no lock use player on page.

So what can I do ? I don't understand your answer.

**Comment by winskil** [comment]
> > For uBlock Origin MV2 and AdGuard MV2:
> > ```
> > kaptainmusic.com##div[class*="CookieCategoryWrapper"]:upward(2)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Pop-up no lock use player on page.
> 
> So what can I do ? I don't understand your answer.

If you have uBlock Origin installed, you can add `kaptainmusic.com##div[class*="CookieCategoryWrapper"]:upward(2)` to your filters
<img width="1920" alt="Screenshot 2022-09-17 at 14 03 04" src="https://user-images.githubusercontent.com/44670482/190853466-84f15ca0-b5c9-45f9-a110-2c7eadbb4bf8.png">
